recipes/protobuf/all/conanfile.py

```diff
@@ -114,10 +114,11 @@ def _configure_cmake(self):
         cmake.definitions["CMAKE_INSTALL_CMAKEDIR"] = self._cmake_install_base_path.replace("\\", "/")
         cmake.definitions["protobuf_WITH_ZLIB"] = self.options.with_zlib
         cmake.definitions["protobuf_BUILD_TESTS"] = False
-        cmake.definitions["protobuf_BUILD_PROTOC_BINARIES"] = True
+        if self.settings.os == "tvOS":
+            cmake.definitions["protobuf_BUILD_PROTOC_BINARIES"] = False
```

Comment:
```suggestion
        cmake.definitions["protobuf_BUILD_PROTOC_BINARIES"] = self.settings.os != "tvOS"
```
Keep previous behavior, but, preserve False when building tvOS